README.md

```diff
@@ -10,11 +10,43 @@ IoniconsAPI plugin
 
 This plugin provides [ionicons](https://ionic.io/ionicons), also known as "symbols", for your Jenkins plugins.
 
+First, add the ionicons-api as dependency to your `pom.xml`:
+
+```xml
```

Comment:
worth linking to the equivalent of https://plugins.jenkins.io/slack/#dependencies?

It gives you a copy paste snippet with the latest version

I didn't link to the one for this as it doesn't appear to be on the plugin site yet

Reply:
> worth linking to the equivalent of https://plugins.jenkins.io/slack/#dependencies?

Absolutely!